Utils/TextureUtils.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+
+namespace DUCK.Utils
+{
+	public static class TextureUtils
+	{
+		public static Texture2D CombineTextures(Texture2D baseTexture, Texture2D addedTexture, int xOffset = 0, int yOffset = 0, bool additive = false)
+		{
+			var basePixels = baseTexture.GetPixels();
```

Comment:
I think we should add some simple comments, to make this easier to follow

Reply:
Done